src/error.js

```diff
@@ -234,6 +239,32 @@ export function isCancellation(errorOrMessage) {
   return false;
 }
 
+/**
+ * Returns an error for component blocked by consent
+ * @return {!Error}
+ */
+export function blockedByConsentError() {
+  return new Error(BLOCK_BY_CONSENT);
+}
+
+/**
+ * @param {*} errorOrMessage
+ * @return {boolean}
+ */
+export function isBlockedByConsent(errorOrMessage) {
+  if (!errorOrMessage) {
+    return false;
+  }
+  if (typeof errorOrMessage == 'string') {
+    return startsWith(errorOrMessage, BLOCK_BY_CONSENT);
+  }
+  if (typeof errorOrMessage.message == 'string') {
+    return startsWith(errorOrMessage.message, BLOCK_BY_CONSENT);
```

Comment:
Why `startsWith` instead of `==` here and above?

Reply:
Ah, it's because of the USER_ERROR_SENTINEL which gets suffixed in user error messages. I guess `cancellation()` is a general purpose error.

You can leave this if you think we'll use consent errors in user error logging.

Reply:
Ah! I see. Thanks Will!